docs/RELEASES.md

```diff
@@ -0,0 +1,23 @@
+# Releases
+
+Telegraf has four minor releases a year in March, June, September, and
+December. In between each of those minor releases, there are 2-4 bug fix
+releases that happen every 3 weeks.
+
+This [Google Calendar][] is kept up to date for upcoming releases dates.
+Additionally, users can look at the [GitHub milestones][] for the next minor
+and bug fix release.
+
+## Versioning
+
+Telegraf uses semantic versioning.
+
+## Minor vs Patch Release
+
+PRs that resolves issues are released in the next release. PRs that introduce
+new features are held for the next minor release. Users can view what
+[GitHub milestones][] a PR belongs to to determine the release it will go out
```

Comment:
There's nothing incorrect here, though "to to" reads kind of poorly to me. What do you think of `Users can view what GitHub milestones a PR belongs to when they want to determine the release it will go out with.` or similar?